index.rst

```diff
@@ -993,17 +994,11 @@ Energy/Solar Management
 
 .. imgtable::
 
-    Remote Receiver, components/remote_receiver, remote.svg, dark-invert
-    Remote Transmitter, components/remote_transmitter, remote.svg, dark-invert
-    Status LED, components/status_led, led-on.svg, dark-invert
-    Display Menu Core, components/display_menu/index, folder-open.svg, dark-invert
-    LCD Menu, components/display_menu/lcd_menu, lcd_menu.png
-    Graphical Display Menu, components/display_menu/graphical_display_menu, graphical_display_menu.jpg
-    Graphical Layout, components/graphical_layout/index, graphical_layout.png
-    Matrix Keypad, components/matrix_keypad, matrix_keypad.jpg
-    Wiegand Reader, components/wiegand, wiegand.jpg
-    HTTP Request, components/http_request, connection.svg, dark-invert
-    mDNS, components/mdns, radio-tower.svg, dark-invert
+    PipSolar-compatible PV Inverter, components/pipsolar, pipsolar.jpg
+    Power Supply, components/power_supply, power.svg, dark-invert
+    Resol VBus, components/vbus, resol_deltasol_bs_plus.jpg
+    SML, components/sml, sml.svg
+    SUN-GTIL2 inverter, components/sun_gtil2, sun_1000g2.png
```

Comment:
??

Reply:
Nerfed merge, fixing.

Reply:
Actually, seems right, this what's in https://github.com/esphome/esphome-docs/blob/74b02f025383386dc3e70d60fa15628598ad7ce5/index.rst?plain=1#L996C1-L1000C61